src/Shared/SemanticConventions.cs

```diff
@@ -55,7 +55,7 @@ internal static class SemanticConventions
     public const string AttributeDbInstance = "db.instance";
     public const string AttributeDbCassandraKeyspace = "db.cassandra.keyspace";
     public const string AttributeDbHBaseNamespace = "db.hbase.namespace";
-    public const string AttributeDbRedisDatabaseIndex = "db.redis.database_index";
+    public const string AttributeDbRedisFlagsKeyName = "db.redis.flags";
```

Comment:
Where is this attribute name coming from ?  I am not able to locate it in semantic conventions : https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/redis.md

Reply:
The attribute already exists in the current tracing implementation. Although, I didn't locate it at the semantic convention, like you said. However, I kept it. Does it make sense to keep it and add to the semantic convention or remove in the implementation (both trace and metrics)?

Reply:
If something does not exist in semantic convention, then its better to remove it. 
(It is recommended to open an issue to track removing it first, and then do the actual removal)

Reply:
issue created: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/2000